i18n/it/building.md

```diff
@@ -178,30 +209,39 @@ $ make install
 - ```make doxygen``` produce documentazione Doxygen
 - ```make clean``` pulisce cartelle build e gli output Doxygen
 - ```make help``` mostra le opzioni di build CMake disponibili
+- ```make uninstall``` uninstall Kovri
 
 #### Note
 - L'output di Doxygen sarà nella cartella```doc```
-- Tutti gli altri output di build saranno nella cartella ```build```
+- Tutti gli altri output di compilazione saranno nella cartella ```build```
 
 ### Clang
 Per eseguire il build con clang, **devi** esportare i seguenti:
 
 ```bash
-$ export CC=clang CXX=clang++  # rimpiazza ```clang``` con una versione/percorso di clang di tua scelta
+$ export CC=clang CXX=clang++  # rimpiazza ```clang``` con una versione/percorso di clang a tua scelta
 ```
 
 ### FreeBSD
 ```bash
-$ export CC=clang36 CXX=clang++36
+$ export CC=clang36 CXX=clang++36 #o una versione di clang più recente
 $ gmake && gmake install
 ```
-- Rimpiazza ```make``` con ```gmake``` per tutte le altre opzioni di build
+- Rimpiazza ```make``` con ```gmake``` per tutte le altre opzioni di compilazione
+
+### OpenBSD
+```bash
+$ export CC=clang CXX=clang++  # clang raccomandato, altrimenti egcc/eg++
+$ gmake && gmake install
+```
+- Rimpiazza ```make``` with ```gmake``` per tutte le altre opzioni di compilazione
+
 
 ### (Opzionale) percorso data personalizzato
 Puoi personalizzare il percorso dei dati di Kovri a tuo piacimento. Semplicemente esporta ```KOVRI_DATA_PATH```; esempio:
 
 ```bash
-$ export KOVRI_DATA_PATH=$HOME/.another-kovri-data-path && make && make install
+$ export KOVRI_DATA_PATH=$HOME/.un altro-percorso-per-kovri && make && make install
```

Comment:
Can we make this a single line? e.g., `.un-altro-percorso-per-kovri`?

Reply:
fixed